okapi/core.php

```diff
@@ -1206,6 +1206,83 @@ public static function get_node_code()
     }
 
     /**
+     * Return the recommended okapi_base_url.
+     *
+     * This is the URL which we want all *new* client applications to use.
+     * OKAPI will suggest URLs with this prefix in various context, e.g. in all
+     * the dynamically generated docs.
+     *
+     * Also see `get_allowed_base_urls` method.
+     */
+    public static function get_recommended_base_url()
+    {
+        return Settings::get('SITE_URL')."okapi/";
```

Comment:
I'm not sure why, but it seems that OCDE seems to be adding it dynamically. Perhaps you're feeding it directly to `SITE_URL` in your `okapi_settings.php`?